src/cljs/main/broadfcui/common/table/table.cljs

```diff
@@ -1,17 +1,17 @@
 (ns broadfcui.common.table.table
   (:require
-   [dmohs.react :as react]
-   [clojure.set :as set]
-   [broadfcui.common.components :as comps]
-   [broadfcui.common.style :as style]
-   [broadfcui.common.table.body :as body]
-   [broadfcui.common.table.column-editor :refer [ColumnEditButton]]
-   [broadfcui.common.table.paginator :refer [Paginator]]
-   [broadfcui.common.table.utils :as table-utils]
-   [broadfcui.persistence :as persistence]
-   [broadfcui.utils :as utils]
-   ))
-
+    [dmohs.react :as react]
+    [clojure.set :as set]
+    [clojure.string :as string]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.style :as style]
+    [broadfcui.common.table.body :as body]
+    [broadfcui.common.table.column-editor :refer [ColumnEditButton]]
+    [broadfcui.common.table.paginator :refer [Paginator]]
+    [broadfcui.common.table.utils :as table-utils]
+    [broadfcui.persistence :as persistence]
+    [broadfcui.utils :as utils]
+    ))
```

Comment:
We(Isaac and I)'re not sure what changed, but now the one space indentation (what's in red here) is right.